ddtrace/utils/__init__.py

```diff
@@ -36,9 +36,9 @@ def integration_service(config, pin, global_service_fallback=False):
 
     # Integrations unfortunately use both service and service_name in their
     # configs :/
-    elif "service" in config:
+    elif "service" in config and config.service is not None:
```

Comment:
This change is necessary because a service will now always be specified as `None` by default.